core/src/packages/requests.ts

```diff
@@ -0,0 +1,138 @@
+import { core } from "../models";
+import { types } from "../types";
+
+const pkg = core.createPackage<any>({name: "HTTP Requests"});
+
+pkg.createNonEventSchema({
+  name: "GET",
+  variant: "Exec",
+  async run({ ctx }) {
+    const response = await fetch(ctx.getInput<string>("url"))
+    
+    // TODO: Change when Objects implemented
+    if (response.ok) {
+      const text = await response.text();
+      ctx.setOutput("response", text);
+    } else {
+      ctx.setOutput("response", "Bad response")
```

Comment:
Avoid using dummy values like this, it should be up to further nodes to not use `response` if `status` isn't a 200